sql/migrations/20221107191704_world.sql

```diff
@@ -225,13 +225,13 @@ UPDATE `quest_template` SET `RewSpell` = 20219  WHERE `entry` = 3641; -- Gnomish
 UPDATE `quest_template` SET `RewSpell` = 20219  WHERE `entry` = 3639; -- Gnomish Engineering (Horde)
 
 -- Correct specialisations teaching artisan of profession
-UPDATE `spell_chain` WHERE `spell_id` = 9787 SET `prev_spell` = 3538 AND `rank` = 4; -- Correct for Weaponsmithing
-UPDATE `spell_chain` WHERE `spell_id` = 9788 SET `prev_spell` = 3538 AND `rank` = 4; -- Correct for Armorsmithing
-UPDATE`spell_chain` WHERE `spell_id` = 10656 SET `prev_spell` = 3811 AND `rank` = 4; -- Correct for Dragonscale leatherworking
-UPDATE `spell_chain` WHERE `spell_id` = 10658 SET `prev_spell` = 3811 AND `rank` = 4; -- Correct for Elemental leatherworking
-UPDATE `spell_chain` WHERE `spell_id` = 10660 SET `prev_spell` = 3811 AND `rank` = 4; -- Correct for Tribal leatherworking
-UPDATE `spell_chain` WHERE `spell_id` = 20219 SET `prev_spell` = 4038 AND `rank` = 4; -- Correct for Gnomish engineering
-UPDATE `spell_chain` WHERE `spell_id` = 20222 SET `prev_spell` = 4038 AND `rank` = 4; -- Correct for Goblin engineering
+UPDATE `spell_chain` WHERE `spell_id` = 9787 SET `prev_spell` = 3538, `rank` = 4; -- Correct for Weaponsmithing
+UPDATE `spell_chain` WHERE `spell_id` = 9788 SET `prev_spell` = 3538, `rank` = 4; -- Correct for Armorsmithing
+UPDATE`spell_chain` WHERE `spell_id` = 10656 SET `prev_spell` = 3811, `rank` = 4; -- Correct for Dragonscale leatherworking
+UPDATE `spell_chain` WHERE `spell_id` = 10658 SET `prev_spell` = 3811, `rank` = 4; -- Correct for Elemental leatherworking
+UPDATE `spell_chain` WHERE `spell_id` = 10660 SET `prev_spell` = 3811, `rank` = 4; -- Correct for Tribal leatherworking
+UPDATE `spell_chain` WHERE `spell_id` = 20219 SET `prev_spell` = 4038, `rank` = 4; -- Correct for Gnomish engineering
+UPDATE `spell_chain` WHERE `spell_id` = 20222 SET `prev_spell` = 4038, `rank` = 4; -- Correct for Goblin engineering
```

Comment:
Why is the WHERE clause before the SET ? Is that even valid ? I've never seen it swapped like this.

Reply:
Yeah obvious mistake on my part